src/harberger-lib/components/Web3Connect.res

```diff
@@ -39,34 +37,9 @@ let make = (~clickAction=() => ()) => {
         ReasonReactRouter.push("#")
         connectWeb3(RootProviderTypes.NoAction)
       }}>
-      {"Connect"->React.string}
+      {"Log In"->React.string}
     </Rimble.Button>
   }
 
   <div className=Styles.loginButton> web3Button </div>
 }
-
-module Modal = {
-  @react.component
-  @dead("Modal.+make") let make = () => {
```

Comment:
When components say `@dead` it means that the dead-code analysis saw no use of that code - ie it can be deleted.

Reply:
`dead-code analysis` <- Is this like a linting tool or packaged into rescript?

Reply:
https://github.com/reason-association/reanalyze - its a pretty cool tool. It just is only for ocaml (or reason/rescript) code, so it doesn't understand places where we are calling out to javascript. But really nice to have something show you where code is 'dead' and can be deleted.

I was mostly just testing it, but I think it has utility in keeping our codebase clean.